ouroboros-consensus/test-storage/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -204,7 +204,7 @@ deriving instance (TestConstraints blk, Eq   it, Eq   rdr)
 deriving instance (TestConstraints blk, Show it, Show rdr)
                => Show (Success blk it rdr)
 
-run :: forall m blk. (MonadSTM m, MonadAsync m)
+run :: forall m blk. (IOLike m)
```

Comment:
Get rid of the parens for consistency?

Reply:
We're not very consistent about parens for single constraints anyway... :/